From: Jan Engelhardt <jengelh@inai.de>
Date: 2012-05-11 08:16:47.544680812 +0200
Upstream: depending on whether it is considered an upstream bug or
	not, it should be sent

Look in the right spot for plugins.

---
 librecad/src/lib/engine/rs_system.cpp |    2 ++
 1 file changed, 2 insertions(+)

Index: LibreCAD-2.0.2/librecad/src/lib/engine/rs_system.cpp
===================================================================
--- LibreCAD-2.0.2.orig/librecad/src/lib/engine/rs_system.cpp
+++ LibreCAD-2.0.2/librecad/src/lib/engine/rs_system.cpp
@@ -642,6 +642,9 @@ QStringList RS_System::getDirectoryList(
     }
     RS_SETTINGS->endGroup();
 
+	if (subDirectory == "plugins")
+		dirList.append("##LIBDIR##" + appDirName + "/" + subDirectory);
+
     QStringList ret;
 
     RS_DEBUG->print("RS_System::getDirectoryList: Paths:");
